src/com/twu/biblioteca/MainMenu.java

```diff
@@ -9,16 +9,18 @@
     private final String welcome = "Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!";
     private final String invalidMessage = "Please select a valid option!";
     private final String goodBye = "Goodbye!";
+    private final String checkOutBookReference = "Enter the number of the book you want to checkout:";
```

Comment:
It's your choice  to pull these strings out into constants or not, but there is a heuristic call the [Rule of 3s](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) which people often use to help them work out when the tipping point has been reached for these types of things.

Reply:
So maybe keep them in their own separate methods? That seems to be where intelliJ is trying to steer me.

Reply:
If they are only used in a single method, then declaring them there (as a local variable) or just inlining the strings is probably a better option as the reader of your code doesn't need to scan outside of the method to understand what `checkOutBookReference` means.

Reply:
Ok thanks, saves the reader from having to jump up and down in the code.